src/spoof.ts

```diff
@@ -236,23 +239,65 @@ export function path (start: Vector, end: BoundingBox | Vector, optionsOrSpread?
   const baseTime = speed * MIN_STEPS
   const steps = Math.ceil((Math.log2(fitts(length, width) + 1) + baseTime) * 3)
   const re = curve.getLUT(steps)
-  return clampPositive(re)
+  return clampPositive(re, optionsResolved)
+}
+
+const clampPositive = (vectors: Vector[], options?: PathOptions): Vector[] | TimedVector[] => {
+  const clampedVectors = vectors.map((vector) => ({
+    x: Math.max(0, vector.x),
+    y: Math.max(0, vector.y)
+  }))
+
+  return options?.showTimestamps === false ? clampedVectors : generateTimestamps(clampedVectors, options)
```

Comment:
Unless you intend timestamps to be included by default, this should be `return options?.showTimestamps ? generateTimestamps(clampedVectors, options) : clampedVectors`

Reply:
Pretty sure I was explicit because the linter complains about it needing to be handled explicitly since it's optional.

Reply:
That's super weird. Well then, either

```jsx
return options?.showTimestamps === true ? generateTimestamps(clampedVectors, options) : clampedVectors
```
or
```tsx
return options?.showTimestamps === false || options?.showTimestamps ===undefined ? clampedVectors : generateTimestamps(clampedVectors, options) 
```

Actually, does it work if you wrap in a `Boolean`?

```tsx
return Boolean(options?.showTimestamps) ? generateTimestamps(clampedVectors, options) : clampedVectors
```

Reply:
The problem with the current `=== false` implementation is that if it's `undefined`, you're generating timestamps